phc-rails/app/models/service.rb

```diff
@@ -18,17 +18,41 @@ class Service < ActiveRecord::Base
   def self.services
     # In the future, we may retrieve these through the Salesforce API, but
     # for now they are hardcoded in
-    medical = ["Addiction_Recovery__c", "Dental__c", "HIV_STI_Testing__c", "Massage__c", \
-               "Medical__c", "Mental_Health__c", "Needle_Exchange__c", "Podiatry__c", \
-               "TB_Testing__c", "Vision_Readers__c", "Vision_Prescription__c"]
-    support = ["Adult_Probation__c", "Banking__c", "Books__c", "CAAP__c", "CalFresh__c", \
+    services = ["Addiction_Recovery__c", "Adult_Probation__c", "Banking__c", "Dental__c", \
+               "Healthy_SF__c", "Massage__c", "Medical__c", "Mental_Health__c", \
```

Comment:
Line is too long. [81/80]<br>Align the elements of an array literal if they span more than one line.